services/notification/.gitignore

```diff
@@ -0,0 +1,244 @@
+
```

Comment:
You can pretty much ignore everything here down. This is a place holder for the notifications stack we don't really support yet.